iotedgedev/azurecli.py

```diff
@@ -226,10 +226,10 @@ def list_resource_groups(self):
 
         return result
 
-    def apply_configuration(self, deviceId, connection_string, config):
+    def apply_configuration(self, deviceId, connection_string, hubName, config):
```

Comment:
I just noticed that the parameter names here are a mix of camelCase and underscores.  Can you make consistent with underscores?

`(self, device_id, connection_string, hub_name, config)`